### DISCUSSION TITLE
Support GoLang Project

### DISCUSSION THREAD

**Discussion by guionardo** [deta/cloud-docs#262] [Ideas] [answered]
A golang API running binary could be very fast and good resource saving App. 
Best than python and JS.

Like the [other](https://github.com/orgs/deta/discussions/226) sugestion about Typescript support, there are special needs to implements this on runtime?

**Comment by cofob** [comment, marked as answer]
Pepperoni, pizza, pasta, Luigi!

I showed in the discord chat how you can do this. In short: put the binary in the folder with the project and run it at each request, and proxy all requests to it.

**Comment by abdelhai** [comment]
hi, thanks for the feedback! we're reworking our runtimes platform at the moment. our goal is to support all languages/framework.
